common/component/kafka/kafka.go

```diff
@@ -229,11 +249,48 @@ func (k *Kafka) Init(ctx context.Context, metadata map[string]string) error {
 	return nil
 }
 
+func (k *Kafka) ValidateAWS(metadata map[string]string) (string, string, string, string, string, error) {
```

Comment:
having a function return so many variables is really smelly

is it possible to return a struct? what struct?

Reply:
I made a deprecated struct like i did for postgres. good call